streaming/src/main/scala/org/apache/spark/streaming/receiver/Receiver.scala

```diff
@@ -271,7 +271,7 @@ abstract class Receiver[T](val storageLevel: StorageLevel) extends Serializable
   }
 
   /** Get the attached executor. */
-  private def executor = {
+  private[streaming] def executor = {
```

Comment:
Why this change? If you want to access private members, its best to use the `PrivateTester` See
https://github.com/apache/spark/blob/master/streaming/src/test/scala/org/apache/spark/streaming/StreamingContextSuite.scala#L866